### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_MenuBar_ScreenReader: Narrator is not announcing Tooltips at all times while tabbing on controls

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#3956]
**Environment Details:**
Application Name:AzureNotebooks_Jyupter
Login Url: notebooks.azure.com
BrowserDetails:
Edge:Microsoft Edge 41.16299.611.0
Operating System:Windows10

**ReproSteps:**
1. Login with the appropriate credentials to azure notebook account.    
2. Navigate to any of the file names present in the samples-->>clickin the file name-->Welcome page opens with menubar.
3. Start Narrator.
4. Navigate till "File" menubar and hit enter.
5. Verify Narrator not reading Tooltip at all times for "Make a Copy" while using Tab key.

**Actual:**
Narrator not reading Tooltip of "Make a copy" at all times while using tab key.
* While first tabbing on control "Make a copy" narrator not announce tooltip, but after using "Shift+Tab" key narrator announce the tooltip.

**Note:**
Same issue is observed in **NVDA** also.
Issue is observed randomly.

**Expected:**
Narrator should announce Tooltip at all times for all controls while using "Tab" and "Shift+Tab" key.

**UserImpact:**
ScreenReader users find difficulty if narrator not announce Tooltip at all time for controls.

**Attachment:**
(https://github.com/jupyter/notebook/files/2399920/A11y_AzureNotebooks_Jupyter_MenuBar_ScreenReader.pptx)

**Comment by jtpio** [comment]
Closing as there are now different issues tracking accessibility improvements for Jupyter Notebook 7, for example https://github.com/jupyter/notebook/issues/6800

Thanks!
